### ISSUE TITLE
Is this assuming that the device and desktop are on the same local network?

### ISSUE THREAD

**Issue by drjasonharrison-vp-eio** [balena-io-experimental/balena-mosh-poc#1]
The readme.md appears to assume that the device and desktop are on the same local network. Is this a correct interpretation? 

What if the device is out in the field? How do we set up the tunnels?

**Comment by afitzek** [comment]
No there is no such assumption, because this uses ngrok to tunnel to the device. Can you point me to the specific text that seems to assume that the device and desktop are on the same local network? Maybe there is some wrong language in the readme.

**Comment by drjasonharrison-vp-eio** [comment]
The example IP addresses for the device and desktop are on the same subnet.

**Comment by afitzek** [comment]
Do you mean the IPs: `10.0.0.100` and `10.0.0.200`? These are the IPs of the virtual p2p ssh tunnel.

**Comment by drjasonharrison-vp-eio** [comment]
Yes, those ip addresses. Because they are on the same subnet I was interpreting them as machine ip addresses.
